packages/market-service/src/exchange-rates-host/exchange-rates-host.test.ts

```diff
@@ -74,3 +75,29 @@ describe('ExchangeRateHostService', () => {
     })
   })
 })
+
+describe('makeUrls', () => {
+  it('should make one url', () => {
+    const start = dayjs('2020-01-01')
+    const end = dayjs('2020-01-02')
+    const symbol = 'EUR'
+    const baseUrl = 'https://api.exchangeratesapi.io'
+    expect(makeUrls(start, end, symbol, baseUrl)).toEqual([
+      'https://api.exchangeratesapi.io/timeseries?base=USD&symbols=EUR&start_date=2020-01-01&end_date=2020-01-02'
+    ])
+  })
+
+  it('should make five urls', () => {
+    const start = dayjs('2017-05-20')
+    const end = dayjs('2022-05-17')
+    const symbol = 'EUR'
+    const baseUrl = 'https://api.exchangeratesapi.io'
+    expect(makeUrls(start, end, symbol, baseUrl)).toEqual([
+      'https://api.exchangeratesapi.io/timeseries?base=USD&symbols=EUR&start_date=2017-05-20&end_date=2018-05-21',
+      'https://api.exchangeratesapi.io/timeseries?base=USD&symbols=EUR&start_date=2018-05-21&end_date=2019-05-22',
+      'https://api.exchangeratesapi.io/timeseries?base=USD&symbols=EUR&start_date=2019-05-22&end_date=2020-05-22',
+      'https://api.exchangeratesapi.io/timeseries?base=USD&symbols=EUR&start_date=2020-05-22&end_date=2021-05-23',
+      'https://api.exchangeratesapi.io/timeseries?base=USD&symbols=EUR&start_date=2021-05-23&end_date=2022-05-17'
```

Comment:
I'm not sure if the response includes data for the date specified in `end_date`, but if it does are we receiving duplicate data back? E.g. 2 requests in this example containing time-series data for `2018-05-21`?